src/ddspdrum/parameter.py

```diff
@@ -0,0 +1,85 @@
+"""
+Parameter Class
+"""
+
+import numpy as np
+
+
+class Parameter:
+    """
+    Parameter class is a structure for keeping track of parameters
+    that have a specific range. Also handles functionality for converting
+    to and from a range between 0 and 1.
+
+    Parameters
+    ----------
+    value (float)   :   initial value of this parameter
+    minimum (float) :   minimum value that this parameter can take on
+    maximum (float) :   maximum value that this parameter can take on
+    scale   (float) :   scaling to apply when converting to and from a value with
```

Comment:
I see. Maybe a more descriptive variable name? `curve` -- and then I would follow suit changing `alpha`

Reply:
I don't completely understand this and need your guys advice: Do we really need the full range of curve values? Maybe we just need strings: "linear", "log", and "exp"?

Agree that the name is confusing

Reply:
I'm into this idea as well, strings.

Reply:
There might be some parameters where were want a more extreme curve. But I am also game to simplify things with strings -- let's go for that.

Reply:
Yeah the nice thing about strings if is we want to add one more, we can easily add it to the switch-statement

Reply:
I think @jorshi's point is that you might want to continuously adjust the severity of the curve. I get that. But I don't know how much of a perceptual difference it will make, and it might save us some headache just to constrict the problem at some point.

Reply:
@maxsolomonhenry You don't adjust `scale` during training, I don't think that's @jorshi 's proposal. These are decided offline by the people implementing the modules (i.e. us) during design time, using expert knowledge about the perceptual phenomena around each parameter. Is my current understanding of the pattern we're using.

Reply:
Ya this wouldn't be updated during training -- but some users might want to be able to adjust the severity of the curve. I think we're fine to stick with log, exp, or linear for the time being.